### ISSUE TITLE
Amiga: Support "VGA Productivity" modes on ECS

### ISSUE THREAD

**Issue by egrath** [emutos/emutos#52]
**THIS IS AN ENHANCEMENT NOT A BUG**

Currently all resolutions above 256 vertical lines are interlaced. However, all Amigas who use ECS Chipset (Amiga 500+, Amiga 600, Amiga 3000) or the later AGA Chipset (Amiga 1200, Amiga 4000) support a non-interlaced resolution of up to 480 vertical lines.

It would be a great enhancement for people running EmuTOS on those machines to have these resolutions available, as it makes working with GEM applications a much more pleasent experience.

**Comment by vinriviere** [comment]
Hum... I have an A500+, and indeed that would be nice. I understand that those modes work on a VGA monitor. Can they also work on a standard TV?

**Comment by egrath** [comment]
No, unfortunately not. When using the "VGA Productivity" mode in an ECS Amiga, the output is a 31.5 Khz signal as opposed to the 15 Khz when using the standard modes the Amiga provides.

**Comment by vinriviere** [comment]
I just found this very nice page about Amiga video modes:
https://amiga.lychesis.net/articles/ScreenModes.html

Sure, there is room to improve EmuTOS with ECS/AGA video modes.
